docs/auth.md

```diff
@@ -0,0 +1,149 @@
+The group assignments can then be used to refine permissions within workspaces with custom role bindings.
+
+Currently the following permissions (verbs) can be declared:
+
+| Verb     | Group                         | Bootstrap cluster rolebinding |
+| -------- | ----------------------------- | ----------------------------- |
+| `admin`  | `system:kcp:workspace:admin`  | `admin`                       |
+| `edit`   | `system:kcp:workspace:edit`   | `cluster-admin`               |
+| `view`   | `system:kcp:workspace:edit`   | `view`                        |
+| `access` | `system:kcp:workspace:access` | N/A                           |
+
+Example 1:
+
+As a user being part of the group `idp:admins` I want to have Kubernetes `admin` permission
+inside workspace `user_workspace1` whose organizational workspace is `org_workspace1`.
+
+The following cluster role and binding need to be declared in the organizational workspace `org_workspace1`:
+
+```
+# defined in the org workspace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: workspace1-admins
+rules:
+  - apiGroups: ["tenancy.kcp.dev"]
+    resources: ["clusterworkspaces/content"]
+    resourceNames: ["user_workspace1"]
+    verbs: ["admin"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: workspace1-idp:admins
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: workspace1-admins
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: idp:admins
+```
+
+Example 2:
+
+As an authenticated user I want to have the same permissions as `system:authenticated`
+in the `default` workspace whose organizational workspace is the `root` workspace.
+
+The following cluster role and binding need to be declared in the `root` workspace:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: default-access
+rules:
+  - apiGroups: ["tenancy.kcp.dev"]
+    resources: ["clusterworkspaces/content"]
+    resourceNames: ["default"]
+    verbs: ["access"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: root:default-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: default-access
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
+```
+
+## Local authorizer
+
+Once the workspace content authorizer has passed successfully, RBAC rules contained
+in the workspace derived from the request context are evaluated.
+
+This authorizer ensures that RBAC rules contained within a workspace are being applied
+and work just like in a regular Kubernetes cluster.
+
+Note that groups added by the workspace content authorizer can be used for role bindings in that workspace.
+
+## Bootstrap policy authorizer
+
+The bootstrap policy authorizer works just as the local authorizer but references RBAC rules
```

Comment:
(No action required) Maybe detail the intended purpose of the bootstrap policy authorizer (e.g. presumably to enable a standard set of initial policies)?